app/main.py

```diff
@@ -2,0 +1,22 @@
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    Cafe.count_friends = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (OutdatedVaccineError, NotVaccinatedError):
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            Cafe.count_friends += 1
+
+    if Cafe.count_friends > 0:
+        return f"Friends should buy {cafe.count_friends} masks"
```

Comment:
The use of `cafe.count_friends` here is incorrect. It should be `Cafe.count_friends` if you intend to use the class attribute. However, it's better to refactor this to use an instance attribute as mentioned earlier.